benchexec/tablegenerator/react-table/src/tests/__snapshots__/Summary.test.js.snap

```diff
@@ -7305,7 +7305,7 @@ exports[`Render Summary for multi-table.diff.html 1`] = `
         <div
          className="cell"
          dangerouslySetInnerHTML={"134"}
-         title="avg: 134, max: 134, median: 134, min: 134, stdev: 0"
+         title="avg: 134, max: 134, median: 134, min: 134, stdev: 0.00"
```

Comment:
@PhilippWendler quick question. As `stdev` is a calculated value, should we format it to match the number of significant digits?

Reply:
Yes, except for columns of type `count` without significant digits.

Reply:
This is now fixed on `master`, though the actual result is now `0.000`.